components/automate-ui/src/app/modules/infra-proxy/client-details/client-details.component.scss

```diff
@@ -96,27 +96,44 @@ img {
     @include base-box-shadow();
     background: $chef-white;
     border-radius: $global-radius;
+    border: 0.5px solid $chef-key-tab;
   }
 
   .key-tab-body {
     display: flex;
     padding: 1.5em;
     background: $chef-white;
-    border: 0.5px solid $chef-key-tab;
+
+    chef-snippet {
+      width: 100%;
+    }
+
   }
 
   .key-tab-body-top {
-    padding: 1em;
+    margin: 0 10px;
     background: $chef-white;
-    border-left: 0.5px solid $chef-key-tab;
-    border-right: 0.5px solid $chef-key-tab;
-    border-top: 0.5px solid $chef-key-tab;
+    border-bottom: 0.5px solid $chef-key-tab;
+    text-align: right;
 
-    p {
-      margin: 0;
-      text-align: right;
-      color: $chef-dark-blue;
+    chef-button:focus {
+      outline: 0.5px solid $chef-primary-bright;
     }
+
+    ::ng-deep {
+      chef-button button:focus {
+        outline: 0.5px solid $chef-primary-bright;
+      }
+    }
+
+    chef-icon:focus {
+      outline: none;
+    }
+
+    span:focus {
+      outline: none;
+    }
```

Comment:
I don't see where these are doing anything.  Looks to me like they can be removed.

Reply:
yeah, good catch, I removed these line.